packages/sign-client/src/constants/engine.ts

```diff
@@ -101,6 +101,18 @@ export const ENGINE_RPC_OPTS: EngineTypes.RpcOptsMap = {
       tag: 1115,
     },
   },
+  wc_sessionAuthenticate: {
+    req: {
+      ttl: ONE_HOUR,
+      prompt: true,
+      tag: 1116,
+    },
+    res: {
+      ttl: ONE_HOUR,
+      prompt: false,
```

Comment:
`prompt` was removed from the spec a while back: https://specs.walletconnect.com/2.0/specs/clients/sign/rpc-methods

Do we still depend on it somehow in the JS implementation?

Reply:
`prompt` is still present in all other methods + types so lets keep it for consistency